src/index.html

```diff
@@ -1,18 +1,44 @@
-<!doctype html>
-<html lang="en">
+<!DOCTYPE html>
+<html lang="en" class="html">
   <head>
-    <meta charset="UTF-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1.0"
-    />
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link
-      rel="stylesheet"
-      href="styles/index.scss"
-    />
+    <link rel="stylesheet" href="./styles/main.scss">
```

Comment:
The `link` tag is referencing a `.scss` file. Browsers cannot directly interpret SCSS files. You need to compile the SCSS into a CSS file and then link to the resulting CSS file instead.